src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -429,5 +433,10 @@ public void Dispose()
             _disposing = true;
             Dispose(true);
         }
+
+        private bool IsTriggerMetadataPopulatedByWorker()
+        {
+            return !string.IsNullOrEmpty(_workerCapabilities.GetCapabilityState(LanguageWorkerConstants.RpcTriggerMetadataPopulated));
```

Comment:
I see this is a pattern that seems to have been established, but just wanted to call out that it's a little strange that we only check if it has a value (!isnullorempty) instead of if the value is "true". I could see a future where someone sets the capability to "false", and that would still resolve as `true`

Reply:
Agreed. That was my initial thought but then thought it would be better to make it consistent with how rest of the language workers use this.